dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -239,6 +321,66 @@ void NavigationViewItem::UpdateVisualState(bool useTransitions)
     if (!m_appliedTemplate)
         return;
 
+    // DisabledStates and CommonStates
+    auto enabledStateValue = L"Enabled";
```

Comment:
This logic looks like it could be useful standalone. Perhaps refactor into a helper method that takes a set of booleans and returns a string ?